src/Mod/Path/PathScripts/PathJob.py

```diff
@@ -110,8 +110,10 @@ def __init__(self, obj, models, templateFile=None):
         obj.addProperty("App::PropertyString", "PostProcessorArgs", "Output", QtCore.QT_TRANSLATE_NOOP("PathJob", "Arguments for the Post Processor (specific to the script)"))
         obj.addProperty("App::PropertyString", "LastPostProcessDate", "Output", QtCore.QT_TRANSLATE_NOOP("PathJob", "Last Time the Job was post-processed"))
         obj.setEditorMode('LastPostProcessDate', 2)  # Hide
-        obj.addProperty("App::PropertyString", "LastPostProcessOutput", "Output", QtCore.QT_TRANSLATE_NOOP("PathJob", "Last Time the Job was post-processed"))
+        obj.addProperty("App::PropertyStringList", "LastPostProcessOutput", "Output", QtCore.QT_TRANSLATE_NOOP("PathJob", "Last Time the Job was post-processed"))
```

Comment:
Why a `StringList` ? The usage hasn't changed neither.

Reply:
This property isn't user accessible.  It's used internally by Path_Sanity when it creates the setup report.  The report contains the filename that was written to disk at last postprocess so the machine operator knows which file to load on the machine.  My oversite when writing Path_Sanity was  that output can be split among multiple files if the user is using WCS in the job.  So a list is correct and is (or will be) accessed by Path_Sanity.  (I'll make sure Path_Sanity is updated and include it with this PR)
Camsim also takes a list of filenames so my was this command could check if the property is populated and use the list of files instead of reposting the project every time.  But that means we have to keep track of whether the output is 'fresh' and that the files are still there.   So I'm not using it that way yet.

Reply:
Reworked this.  Job no longer tracks the lastpostprocessed time/file.   
Added a property to the job to optionally generate setupreport with post.